frontend/src/Components/Loader/Loader.jsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+
+// Our Fake Data - Will be changed when we get our endpoint and data from there
+
+const Loader = () => {
+	return (
+		// Loader animation with Text - Can be changed whenever, however
+		<div className="container-fluid d-flex flex-column justify-content-center align-items-center vh-100">
+			{/* Main Loader Style */}
+			<div className="spinner-border text-primary" style={{ width: "5rem", height: "5rem" }} role="status">
+				<span className="sr-only">Loading...</span>
+			</div>
+
+			{/* This is the optional Loader that we may use. */}
+			{/* <div className="spinner-grow" style={{ width: "5rem", height: "5rem" }} role="status">
```

Comment:
Remove commented code, also review other comments if they're making sense like the one on line 3.